packages/react-instantsearch/src/components/Hits.js

```diff
@@ -1,33 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from './classNames.js';
+import BaseWidget from './BaseWidget';
 
-const cx = classNames('Hits');
+const widgetClassName = 'Results';
+const cx = classNames(widgetClassName);
 
-const Hits = ({ hits, hitComponent: HitComponent }) => (
+const Hits = ({ hits, hitComponent: HitComponent, header, footer }) => (
   // Spread the hit on HitComponent instead of passing the full object. BC.
   // ex: <HitComponent {...hit} key={hit.objectID} />
-  <div {...cx('root')}>
-    {hits.map(hit => <HitComponent key={hit.objectID} hit={hit} />)}
-  </div>
+  <BaseWidget widgetClassName={widgetClassName} header={header} footer={footer}>
+    <ul {...cx(['list'])}>
+      {hits.map(hit => <HitComponent key={hit.objectID} hit={hit} />)}
```

Comment:
Maybe we can wrap the `HitComponent` in a `li` because otherwise the user will always need to render his component in a `li` in order to have a valid DOM. We will also be able to add the `className` item on it.